python/deltalake/table.py

```diff
@@ -457,6 +457,7 @@ def create(
         configuration: Optional[Mapping[str, Optional[str]]] = None,
         storage_options: Optional[Dict[str, str]] = None,
         custom_metadata: Optional[Dict[str, str]] = None,
+        raise_if_key_not_exists: Optional[bool] = True,
```

Comment:
I would keep it required

Reply:
Yeah, fair! Should I move it further up in the signature, then, i.e. before the Optional ones?

Reply:
or wait, I'll still use the default True, so it should be fine

Reply:
Best to keep it aligned across :)

Reply:
Not sure what you mean by that 🤔  Had to move it further up in `create_deltalake`: The Rust code was failing to compile because of the ambigious ordering (it wants non-optional before optional ones).